tests/test_models.py

```diff
@@ -133,6 +133,16 @@ def test_save_specific_datetime(self):
         k = BarTime.objects.filter(datetime='1398-04-31 12:12:12')
         self.assertEqual(k[0].datetime.day, 31)
 
+    def test_date_lookup_filter(self):
+        jdatetime_ = jdatetime.datetime(1378, 10, 10)
+        BarTime.objects.create(name='Test', datetime=jdatetime_)
+        count_jdatetime = BarTime.objects.filter(datetime__date=jdatetime_).count()
```

Comment:
let's test the query result directly with the object:
```suggestion
        bartime = BarTime.objects.create(name='Test', datetime=jdatetime_)
        self.assertEqual(BarTime.objects.get(datetime__date=jdatetime_), bartime)
```

Please do it for the other cases in this test as well.